### PR TITLE
User location.

### PR DESCRIPTION
**Pull request by amaury901130**
### Description
-Save user location.
-Get user latest location.

### CODE REVIEW

File: app/controllers/api/v1/locations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1\
```
Comment (TimoPeraza) [inline app/controllers/api/v1/locations_controller.rb]: remove the "\" character

File: app/models/location.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#  user_id    :integer          not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_locations_on_user_id  (user_id)
+#
+
+class Location < ApplicationRecord
```
Comment (TimoPeraza) [inline app/models/location.rb]: I think you have redundant information between this model and the target model. I think the best way is remove from the target model the latitude, the longitude and add a foreign key to the location model and remove the user from the location. WDYT?

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'locations/index'
```
Comment (TimoPeraza) [inline config/routes.rb]: remove these gets

File: app/models/location.rb (hunk 2 above)
Comment (TimoPeraza) [inline app/models/location.rb]: Other thing, i remember that you will need the area for check if exist a match between users, so maybe you can add that attribute to the model.

Comment (amaury901130) [reply app/models/location.rb]: Thi's the location for the user.

Comment (amaury901130) [reply app/models/location.rb]: The area of the target. OK nex PR 😁

**Pull request merged**